1/backend/src/todos.spec.js

```diff
@@ -73,44 +96,70 @@ const FINISH_TODO = gql`
    		}
     }`;
 
+const ADD_USER = gql`
+	mutation AddUser{
+		addUser(login: "milan", password: "password")
+    }`;
+
+const DELETE_USER = gql`
+	mutation DeleteUser{
+		deleteUser(login: "milan", password: "password")
+    }`;
+
+const ASSIGN_USER_TO_TODO = gql`
+	mutation Assign($user: String!, $id: ID!){
+   		assignTodoToUser(user: $user, id: $id) 
+    }`;
+
+
 
 describe('Get Todo', () => {
     it("Receives all Todos", async () => {
-        const todo = await mutate({mutation: GET_TODOS})
-        expect(todo.data).toMatchObject(
-            {
-                "todos": [
-                    {
-                        "message": "Einkaufen",
-                        "completed": false,
-                        "id": "1"
-                    },
-                    {
-                        "message": "Apollo Server aufsetzen",
-                        "completed": false,
-                        "id": "2"
-                    }
-                ]
-            }
-        )
+        const todo = await query({query: GET_TODOS})
+        expect(todo.data).toHaveProperty("todos")
+    })
+    it("Receives one todo (LIMIT 1)", async () => {
```

Comment:
```suggestion
    it("receives one todo (LIMIT 1)", async () => {
```
It's a sentence in third-person starting with `it`